src/BaywatchOperation.php

```diff
@@ -53,7 +53,8 @@ public function enable_preview() {
     if ($consumers) {
       /** @var \Drupal\consumers\Entity\Consumer $consumer */
       $consumer = reset($consumers);
-      $consumer->set('uuid', 'dc881486-c14a-4b92-a0d0-e5dcd706f5ad')->save();
+      $consumer->set('uuid', 'dc881486-c14a-4b92-a0d0-e5dcd706f5ad');
+      $consumer->set('client_id', 'dc881486-c14a-4b92-a0d0-e5dcd706f5ad')->save();
```

Comment:
Not sure if this is because of a change to the `consumer` module but have added to get the consumers page working.